tests/integrations/sanic/test_sanic.py

```diff
@@ -34,7 +40,11 @@ def test_request_data(sentry_init, app, capture_events):
     assert response.status == 200
 
     (event,) = events
-    assert event["transaction"] == "hi"
+    # Sanic versions above and including 21.3 append the app name to the route name
```

Comment:
Should this be the default? I think we should strip the app name so transactions stay consistent in between updates to the Sanic integration.

Reply:
you are right! resolved